### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by presspossum**
Adds a simpler way of adding Python to the system path in Windows.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +65,8 @@ pipenv install
     - If you see `Python 3.8.0` or higher, skip to step 4
     - If you see `Python 3.7.9` or lower, skip to step 2
     - If you see `python : The term 'python' is not recognized`, continue to step 2
-2. Install the latest [Python 3.8](https://www.microsoft.com/en-us/p/python-38/9mssztt1n39l) for Windows
+2. Install the latest [Python 3.8](https://www.python.org/ftp/python/3.8.6/python-3.8.6-amd64.exe) for Windows
```
Comment (presspossum) [inline README.md:68]: Changed to the direct link for installing Python onto Windows from python.org

File: README.md [hunk 2]
```diff
@@ -65,7 +65,8 @@ pipenv install
...
+    - Click `Add Path` at the bottom of the first installation page.
```
Comment (presspossum) [inline README.md:69]: Simplifies the installation path of python so that the user does not have to do step #3

File: README.md [hunk 3]
```diff
@@ -65,7 +65,8 @@ pipenv install
     - If you see `Python 3.8.0` or higher, skip to step 4
     - If you see `Python 3.7.9` or lower, skip to step 2
     - If you see `python : The term 'python' is not recognized`, continue to step 2
```
Comment (Ataraksia) [inline README.md:67]:
```suggestion
    - If you see `Python 3.7.9` or lower, skip to step 2
    - If you see `python : The term 'python' is not recognized`, continue to step 2
```

Comment (Ataraksia) [inline README.md:67]:
```suggestion
    - If you see `Python 3.7.9` or lower, skip to step 2
    - If you see `python : The term 'python' is not recognized`, there are two options.  If you know you have Python 3.8 
       installed, then you may skip to step 3 in order to add it to your PATH.  However, if you are not sure, or would 
       prefer a simpler option, you can simply continue to step 2.
```

File: README.md (hunk 2 above)
Comment (Ataraksia) [inline README.md:69]:
```suggestion
    - Click `Add Path` at the bottom of the first installation page.
    - After the installation completes, you can skip to step 4.
```

File: README.md (hunk 3 above)
Comment (Ataraksia) [inline README.md:67]:
```suggestion
    - If you see `python : The term 'python' is not recognized`, you can either skip to step 3 if you know that you have Python 3.8 installed in order to add it to your PATH or you may continue to step 2 if you are unsure or simply would prefer an easier option.
```

Comment (Ataraksia) [inline README.md:67]:
```suggestion
    - If you see `python : The term 'python' is not recognized`, you may continue to step 2.  Alternatively, if you believe Python 3.8 is installed already, you may skip to step 3 if you wish to simply add it to your PATH manually.
       
```

File: README.md (hunk 2 above)
Comment (krcm0209) [reply README.md:71]: Step 3 shouldn't be necessary anyway, it's more about cleaning up old Python stuff on the path. The Windows Store installation adds itself to the PATH automatically without having to check a box.

Comment (Ataraksia) [reply README.md:71]: I definitely agree.  I considered editing this to remove it entirely, but it seemed a bit of a shame to remove all that hard work that someone typed out in 3 so the changes I made were to try and accommodate keeping it while still informing users of how to go a simpler route in the first place

**Comment by philippnormann** [comment]
Thanks for the suggestion! The python.org installer can indeed prevent most `PATH` related issued automatically :+1:

**Pull request merged**